README.md

```diff
@@ -29,6 +29,55 @@ Run:
 ./index.js couch-import --source-couch-db=[couch-db-url]/_changes --target-registry=[target-registry-url] --shared-fetch-secret=[password-from-console]
 ```
 
+## Importing From npm Orgs
```

Comment:
Suggestion for reordering:

1. Create an authorization token on npmjs.com
2. Create a text file containing the packages you wish to migrate
3. Log in to your npm Enterprise instance
4. Create a matching organization in npm Enterprise
5. Run the migration tool